server.js

```diff
@@ -218,11 +218,33 @@ app.get('/live/:year/:week', function(req, res) {
 	});
 })
 
+/**
+ * GET route for the current week in the season. This route can be used when
+ * wanting to know the current season and the current week in the season.
+ */
+ app.get('/currentSeasonAndWeek', function(req, res) {
+	 url = 'http://www.nfl.com/schedules';
+	 request(url, function(error, response, html) {
+		 if (!error) {
+			 console.log(response.statusCode + ' GET ' + url);
+			 var path = response.request.uri.path;
+			 var resultStrings = path.split("/");
+			 var year = resultStrings[2];
+			 var week = resultStrings[3];
+			 week = week.slice(3, week.length);
+			 res.statusCode = 200;
+			 res.send({'year': year, 'week': week});
```

Comment:
Do you think we could then expand this to simply use its own method to return the data for that current week? Or maybe pass it a flag if you want it to get those results? Basically, give it a url-encoded param that says something like `getResults`:

 - if it's false, simply return what you've got here.
 - if it's true, take your end result, and make a request to itself to get the data for that week and year, and then return that result.

I thought of this only because it could cut down on the amount of work the client has to do to handle a response from this and then situationally do something new, and you'd always get the result you're looking for at a bare minimum.